prow/cmd/tot/main.go

```diff
@@ -79,7 +80,19 @@ func (s *store) save() error {
 func (s *store) vend(b string) int {
 	s.mutex.Lock()
 	defer s.mutex.Unlock()
-	n := s.Number[b] + 1
+	n, ok := s.Number[b]
+	if !ok {
+		if out, err := exec.Command("gsutil", "cat", fmt.Sprintf("gs://kubernetes-jenkins/logs/%s/latest-build.txt", b)).Output(); err != nil {
```

Comment:
could we gate this behind a flag? or if nothing else, make the bucket uri configurable